conference/react-js/app/src/ComponentA.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Why you are pushed ```src/ComponentA.jsx``` and ```src/ComponentA/...```? Please remove this file from ```src``` folder or move to ```ComponentA``` folder.